application/views/quotas/_form.php

```diff
@@ -33,8 +31,10 @@
             <div class=''>
                 <?php echo $form->dropDownList($oQuota, 'action',
                     [
-                        Quota::ACTION_TERMINATE         => gT("Terminate survey"),
-                        Quota::ACTION_CONFIRM_TERMINATE => gT("Allow user to modify their last answers before terminating the survey."),
+                        Quota::TERMINATE_AFTER_ALL_VISIBLE_QUOTA_QUESTION      => gT("Terminate after related visible question was submitted"),
+                        Quota::TERMINATE_AFTER_ALL_QUOTA_QUESTIONS             => gT("Terminate after related visible and hidden questions were submitted"),
```

Comment:
The result are _after related visible and hidden questions step_ , but logically it's TERMINATE_WITH_HIDDEN_QUESTION.

I'm totally unsure on the var name here (mine is worst than your).